tests/src/test/scala/actionContainers/ActionContainer.scala

```diff
@@ -135,19 +135,32 @@ object ActionContainer {
       case (k, v) => s"-e ${k}=${v}"
     } mkString (" ")
 
-    // We create the container...
-    val runOut = awaitDocker(s"run --name $name $envArgs -d $imageName", 10 seconds)
-    assert(runOut._1 == 0, "'docker run' did not exit with 0: " + runOut)
+    // We create the container... and find out its IP address...
+    def createContainer(portFwd: Option[Int] = None) = {
+      val runOut = awaitDocker(
+        s"run ${portFwd.map(p => s"-p $p:8080").getOrElse("")} --name $name $envArgs -d $imageName",
```

Comment:
@rabbah I thought we try at one point to remove the proxy but we ran into trouble, and that's we have the proxy.

can't recall now why didn't push forward for a solution.

But this would be great if we can get rid off the proxy.